proximal/utils/utils.py

```diff
@@ -51,9 +51,12 @@ def _stack(arrays, axis=0):
 # Image utils
 ###############################################################################
 
-def get_test_image(WIDTH):
-    I = cv2.resize(ascent().astype(np.float32), (WIDTH, WIDTH))
-    I /= 255.
+def get_test_image(WIDTH, color=False):
+    if not color:
+        I = cv2.resize(ascent().astype(np.float32), (WIDTH, WIDTH))
```

Comment:
S

Reply:
You are correct. Thank you for spotting the error.